packages/meditrak-app/app/navigation/Navigator.js

```diff
@@ -70,8 +70,8 @@ const config = {
   initialRouteName: INITIAL_SCREEN_NAME,
   initialRouteParams: { surveyScreenIndex: 0 },
   headerMode: 'float',
-  navigationOptions: ({ navigation }) => ({
-    headerLeft: HeaderLeftButton,
+  defaultNavigationOptions: ({ navigation }) => ({
```

Comment:
`navigationOptions` is now changed to `defaultNavigationOptions` in StackNavigatorConfig... (react-stack-navigator documentation was not really clear about this too)